### PR TITLE
exempt 3 call method chaining

### LINKED ISSUE #36: Cannot read property 'callee' of undefined
I reproduced the failure [here](https://github.com/efx/ember-test-helpers-codemod/commit/d86475ae8d63d8e3547f640ab96934984a2b8313).
Frankly, this is 1 integration test out of 217 so I am going to refactor it. 
 I see 2 fixes:
1. add a `filter` with safety check before `replaceWith` [here](https://github.com/simonihmig/ember-test-helpers-codemod/blob/master/transforms/integration/transforms/set-value.js#L107). Downside is that this would leave a legacy pattern, perhaps we inform user and skip the file.
2. refactor that expression to handle this case.

I'm lazy and unmotivated, so I'd vote for 1. I can PR if wanted.

### PR DESCRIPTION
**Pull request by efx**
fix #36

### CODE REVIEW

File: transforms/integration/transforms/set-value.js [hunk 1]
```diff
@@ -67,6 +68,16 @@ function hasFluentTriggerCall(j, path) {
     ;
 }
 
+function hasFluentTriggerAndMoreCall(j, path) {
+  const greatGrandParent = path.parentPath.parentPath.parentPath.node;
```
Comment (efx) [inline transforms/integration/transforms/set-value.js]: used `path.parentPath` as I think it allows safe access when traversing up the tree.

File: transforms/integration/transforms/set-value.js [hunk 2]
```diff
@@ -67,6 +68,16 @@ function hasFluentTriggerCall(j, path) {
...
+  return greatGrandParent
+    && j.MemberExpression.check(greatGrandParent)
+    // add additional method names to the following
+    // to exempt them from processing e.g. focus()
+    && /^focusout|blur$/i.test(greatGrandParent.property.name)
```
Comment (simonihmig) [inline transforms/integration/transforms/set-value.js]: Do we actually have to check this? For one thing there could be other cases of additonal fluent calls besides `focusout` and `blur`. 

I am not sure exactly at which line of code where you were hitting the exception reported in #36, but my understanding was that any additional fluent call after `.trigger()` will lead to this, doesn't it? So would just checking `j.MemberExpression.check()`  be enough here, and cover more invalid cases?

I might miss something though, not fully understanding the problem eventually...

Comment (efx) [reply transforms/integration/transforms/set-value.js]: We probably don't need to check for this. I wanted to be cautious and whitelist the keywords. I'll remove it.
The line I hit the exception was the 3 call chain [here](https://github.com/simonihmig/ember-test-helpers-codemod/pull/37/files#diff-e154e411c6db60dfd89af0762c42ed6dR19).

**Comment by efx** [comment]
@simonihmig any objections to merging this?

**Comment by efx** [comment]
@simonihmig Keine Ursache! I had forgotten it until now 😄

**Pull request merged**